biz.aQute.launcher/src/aQute/launcher/plugin/ProjectLauncherImpl.java

```diff
@@ -343,6 +343,19 @@ public Jar executable() throws Exception {
 				"SHA-1", "MD-5"
 			});
 		jar.setManifest(m);
+
+		String moduleInstruction = getProject().getProperty(Constants.MODULE);
+		if (moduleInstruction != null) {
+			String name = jar.getName();
+			@SuppressWarnings("resource") // don't close the jar
+			Builder b = new Builder();
```

Comment:
You need to close the Builder! So you will need to remove the jar from the builder via `removeClose(jar)` before closing the builder.